MSAL/src/native_auth/network/responses/validator/sign_in/MSALNativeAuthSignInResponseValidator.swift

```diff
@@ -51,8 +51,8 @@ final class MSALNativeAuthSignInResponseValidator: MSALNativeAuthSignInResponseV
                 MSALLogger.log(
                     level: .error,
                     context: context,
-                    format: "SignIn Challenge: Error type not expected, error: \(signInChallengeResponseError)")
-                return .error(.invalidServerResponse)
+                    format: "signin/challenge: Error was not decoded properly, error: \(signInChallengeResponseError)")
```

Comment:
"Unable to decode error response" would be my preferred log message for these errors

Reply:
Done